sdk/textanalytics/ai-text-analytics/test/public/utils/recordedClient.ts

```diff
@@ -75,7 +75,12 @@ export function createClient(
   return new TextAnalyticsClient(
     env.ENDPOINT || "https://dummy.cognitiveservices.azure.com/",
     credential,
-    options
+    /**
+     * The service's rate limit is sometimes reached for the beginAnalyzeActions tests
+     * because of the many requests the tests send. It appears that retrying only 3 times
+     * (the default) is not enough so this logic increases that to 10.
+     */
+    { ...options, retryOptions: { maxRetries: 10 } }
```

Comment:
after increasing the policy default to 10, this might not be needed but perhaps it could be left for documentation of the issue?

Reply:
hmm was this something that took a long time to root cause? Maybe cleaning up the comment would be enough?

Reply:
Fair enought! I removed the option setting, edited the comment, and moved it to the tests.